src/blockdata/transaction.rs

```diff
@@ -220,12 +221,204 @@ impl Default for TxIn {
         TxIn {
             previous_output: OutPoint::default(),
             script_sig: Script::new(),
-            sequence: u32::max_value(),
+            sequence: Sequence::MAX,
             witness: Witness::default(),
         }
     }
 }
 
+/// Bitcoin transaction input sequencing number.
+/// 
+/// The sequence field is used for:
+/// - Indicating whether [BIP-65] absolute lock-times are enabled.
```

Comment:
Just realized this is confusing because lock times are ancient and the behavior was implemented before BIPs existed. BIP-65 is somewhat related but not the entire specification.

I suggest removing the reference from that phrase and perhaps put below something like "Note that transactions spending an output with `OP_CHECKLOCKTIMEVERIFY` MUST NOT use `Sequence::MAX` for the corresponding input. ([BIP-65](https://github.com/bitcoin/bips/blob/master/bip-0065.mediawiki))."

Also "absolute" is not common terminology, so maybe write "Indicating whether the absolute lock-time (specified in `lock_time` field of `Transaction`) is enabled." Singular because there's only one lock time for transaction.